x/nft/simulation/decoder_test.go

```diff
@@ -21,7 +22,10 @@ var (
 )
 
 func TestDecodeStore(t *testing.T) {
-	cdc := simapp.MakeTestEncodingConfig().Codec
+	cfg, err := network.DefaultConfigWithAppConfig(testutil.AppConfig)
```

Comment:
We should _only_use `network` when we're running actual network tests. It should be possible to just do this:
```suggestion
  var cdc codec.Codec
  depinject.Inject(testutil.AppConfig, &cdc)
```